core/src/main/java/org/sonarsource/sonarlint/core/WsHelperImpl.java

```diff
@@ -43,13 +44,15 @@
 import org.sonarsource.sonarlint.core.util.StringUtils;
 import org.sonarsource.sonarlint.core.util.ws.WsResponse;
 
+// TODO
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.core%3Asonarlint-core-parent&branch=junk2&issues=AWEyAmflzwlWAzALxAc0&open=AWEyAmflzwlWAzALxAc0)

[ISSUE_KEY]: AWEyAmflzwlWAzALxAc0